modules/services/redshift-gammastep/lib/options.nix

```diff
@@ -26,13 +26,31 @@ in {
   in [
     (mkRemovedOptionModule [ "services" moduleName "extraOptions" ]
       "All ${programName} configuration is now available through services.${moduleName}.settings instead.")
-    (mkRenamed [ "brightness" "day" ] "brightness-day")
-    (mkRenamed [ "brightness" "night" ] "brightness-night")
   ];
 
   options = {
     enable = mkEnableOption programName;
 
+    brightness = {
+      day = mkOption {
+        type = types.str;
+        default = "1";
```

Comment:
```suggestion
        type = with types; nullOr (either str float);;
        default = null;
```

Reply:
Ah, makes sense. Otherwise these options will be written out regardless. I'll update the MR

Reply:
Would be good to also add an example. Sadly using float literals doesn't show in manual AFAIK, so you probably need to use `literalExample`.